patch.c

```diff
@@ -102,7 +101,7 @@ int patch_execute(struct qdl_device *qdl, int (*apply)(struct qdl_device *qdl, s
 	int ret;
 
 	for (patch = patches; patch; patch = patch->next) {
-		if (strcmp(patch->filename, "DISK"))
+		if (xmlStrcmp(patch->filename, (const xmlChar *)"DISK"))
```

Comment:
can you justify why/how xmlStrcmp() is better than strcmp()? it would have been a good idea to provide that info in the commit message, btw.

Reply:
It's better because XML API returns xmlChar which is unsigned char. As we use libxml2 there is no reason to use normal strcmp.

Reply:
I forgot to mention that the standard does not specify if plain char is signed or unsigned.

Reply:
I have updated patch with commit message with all reasons.